parsers/NTESMO.py

```diff
@@ -130,26 +152,32 @@ def get_data(
 
 def parse_consumption(
     raw_consumption: pd.DataFrame,
-    target_datetime: datetime,
-    logger: Logger,
+    target_datetime: Optional[datetime] = None,
+    logger: Logger = getLogger(__name__),
```

Comment:
This function should always inherit the datetime and logger from the calling function so there is no need to set a default or adding a optional type here.